.env.example

```
@@ -188,6 +189,13 @@ AURORA_CACHE_TOKEN_IN_REDIS=false
 GH_OAUTH_CLIENT_ID=
 GH_OAUTH_CLIENT_SECRET=
 
+# GitHub App (alternative to OAuth, see server/connectors/github_connector/SETUP_GITHUB_APP.md)
+GITHUB_APP_ID=
+GITHUB_APP_CLIENT_ID=
+GITHUB_APP_WEBHOOK_URL=
+GITHUB_APP_SETUP_URL=
+GITHUB_APP_WEBHOOK_SECRET=  # fallback if not in Vault
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)